UICatalog/Scenarios/LineViewExample.cs

```diff
@@ -9,80 +9,82 @@ namespace UICatalog.Scenarios;
 [ScenarioCategory ("Borders")]
 public class LineViewExample : Scenario
 {
-    public override void Setup ()
+    public override void Main ()
     {
-        Win.Title = GetName ();
-        Win.Y = 1; // menu
-        Win.Height = Dim.Fill (1); // status bar
+        // Setup - Create a top-level application window and configure it.
+        Toplevel appWindow = new ();
```

Comment:
Please add `Application.Init ();` into the scenario to avoid exception been throwing.